swap_meet/vendor.py

```diff
@@ -1,2 +1,60 @@
-class Vendor:
-    pass
+from swap_meet.item import Item
+
+
+class Vendor():
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
+    def get_by_category(self, category):
+        things = []
+        for item in self.inventory:
+            if item.category == category:
+                things.append(item)
+        return things
+    
+    def swap_items(self, swapper, my_item, their_item):
+        if my_item in self.inventory and their_item in swapper.inventory:
+            self.remove(my_item) 
+            swapper.add(my_item)
+            swapper.remove(their_item)
+            self.add(their_item)
+            return True
+        else:
+            return False
+    
+    def swap_first_item(self, swapper):
+        if len(self.inventory) is not 0 and len(swapper.inventory) is not 0:
+            thing_one=self.inventory.pop(0)
+            thing_two=swapper.inventory.pop(0)
+            self.add(thing_two)
+            swapper.add(thing_one)
+            return True
+        else:
+            return False
```

Comment:
You already have a function that will do this work for you! `swap_items()` can be called here and then you would pass in the first item of the vender and the other vendor as arguments 
`  return self.swap_items(other, self.inventory[0], other.inventory[0])`